examples/initial_topography/get_Init_Topo.jl

```diff
@@ -0,0 +1,101 @@
+# ~/~ begin <<docs/src/initial-topography.md#examples/initial_topography/get_init_topo.jl>>[init]
```

Comment:
the filename is a bit weird: you mix snake-case and camel-case. In general we have camel case (e.g. `CamelCase`) for filenames that are part of the CarboKitten distribution, while examples are stored in lower case (`snake_case`).